examples/spring-web/README.md

```diff
@@ -11,3 +13,12 @@ A Spring Boot example application to show how to use Bugsnag in a Spring web bas
 - Cause a crash
 
     http://localhost:8080/
+
+## Running in Docker
+
+Execute the following commands in the root of the repository:
+
+```
+docker-compose build
+docker-compose up spring-web
```

Comment:
this seems to work well except for the shutdown command which doesn't end the container. The output is below:

```
spring-web_1  | 2018-08-06 20:05:06.056  INFO 81 --- [nio-8080-exec-6] c.b.e.s.web.ApplicationRestController    : Shutting down application
spring-web_1  | 2018-08-06 20:05:06.057  INFO 81 --- [nio-8080-exec-6] ationConfigEmbeddedWebApplicationContext : Closing org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@48ae9b55: startup date [Mon Aug 06 20:04:50 GMT 2018]; root of context hierarchy
spring-web_1  | 2018-08-06 20:05:06.058  INFO 81 --- [nio-8080-exec-6] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
spring-web_1  | 2018-08-06 20:05:06.059 DEBUG 81 --- [nio-8080-exec-6] com.bugsnag.Bugsnag                      : Closing connection to Bugsnag
```